$OpenBSD: patch-panels_meson_build,v 1.3 2020/11/14 11:03:46 ajacoutot Exp $

No GUdev on OpenBSD (XXX add HAVE_GUDEV to meson_options.txt)
No colord (brings GUdev and GUsb) on OpenBSD (XXX add HAVE_COLORD to meson_options.txt)

Index: panels/meson.build
--- panels/meson.build.orig
+++ panels/meson.build
@@ -4,7 +4,6 @@ panels = [
   'applications',
   'background',
   'camera',
-  'color',
   'datetime',
   'default-apps',
   'diagnostics',
@@ -14,7 +13,6 @@ panels = [
   'location',
   'lock',
   'microphone',
-  'mouse',
   'notifications',
   'online-accounts',
   'power',
